### PR TITLE
Remove functions with wrong endpoints

### PR DESCRIPTION
**Pull request by haardikdharma10**
### Description

Remove functions with wrong endpoints and delete `application_size.go`

### CODE REVIEW

File: application.go [hunk 1]
```diff
@@ -169,7 +169,7 @@ func (c *Client) CreateApplication(config *ApplicationConfig) (*Application, err
 }
 
 // UpdateApplication updates an application
-func (c *Client) UpdateApplication(id string, application *UpdateApplicationRequest) (*Application, error) {
+func (c *Client) UpdateApplication(id string, application *ApplicationConfig) (*Application, error) {
```
Comment (RealHarshThakur) [inline application.go:172]: Why is this changed ? It should remain `UpdateApplicationRequest`

**Review by RealHarshThakur — CHANGES_REQUESTED** [review]

File: application.go (hunk 1 above)
Comment (haardikdharma10) [reply application.go:172]: IIRC, there were some issues I faced with it in the CLI and hence changed it to `*ApplicationConfig`
Also, to be consistent with https://github.com/civo/civogo/blob/master/kubernetes.go#L248 and https://github.com/civo/civogo/blob/master/loadbalancer.go#L154 

I have also changed CLI to use the config. Is there a problem changing this?

Comment (RealHarshThakur) [reply application.go:172]: Yes, the update endpoint needs the fields present in UpdateApplicationRequest. You have to change it back and figure out the issue in CLI/API

Comment (haardikdharma10) [reply application.go]: Okay, updated this PR. PTAL.

**Review by RealHarshThakur — APPROVED** [review]
